modules/k3s/k3s.go

```diff
@@ -0,0 +1,137 @@
+package k3s
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/go-connections/nat"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	// containerPorts {
+	defaultKubeSecurePort     = "6443/tcp"
+	defaultRancherWebhookPort = "8443/tcp"
+	// }
+	defaultKubeConfigK3sPath = "/etc/rancher/k3s/k3s.yaml"
+)
+
+// K3sContainer represents the K3s container type used in the module
+type K3sContainer struct {
+	testcontainers.Container
+}
+
+// RunContainer creates an instance of the K3s container type
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*K3sContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image: "docker.io/rancher/k3s:latest",
```

Comment:
@mritunjaysharma394 I merged but did not notice the `latest` tag. Could you send a follow-up PR updating to a fixed version? 🙏

Reply:
Probably using `v1.27.1-k3s1` would be a good thing

Reply:
Thanks a lot and apologies for missing on it too, I used it for testing and never changed/fixed it, doing it right away

Reply:
Created PR #1225, thanks for the review and the patience and the merge!